### ISSUE TITLE
When a playlist has any new songs, entire playlist is downloaded again

### ISSUE THREAD

**Issue by mathiasfoster** [digitalec/deemon#37]
As the title describes. Expected behaviour is for only the new songs to download, not the entire playlist.

I download to `/Music/`, and then move files to another storage device once downloading has finished.
At the start of every execution of `deemon refresh`, the `/Music/` directory is entirely empty.

Commands/command output/logs have been edited slightly for clarity.

```
> deemon -V
deemon 1.3.1

> deemon refresh
Playlist 'Hot New Rock' has 1 new track(s)
----------------------------
Sending 1 release(s) to deemix for download:
+ Hot New Rock (playlist)...

> cd /Music/Hot\ New\ Rock && ls
... entire playlist displays, 65 songs...
```

In the logs:
```
2021-09-22 08:37:04 [DEBUG] deemon.app.db: Database version 2.2
2021-09-22 08:37:04 [DEBUG] deemon.app.db: Database version 2.2
2021-09-22 08:37:04 [DEBUG] deemon.app.refresh: Starting refresh...
2021-09-22 08:37:04 [DEBUG] deemon.app.refresh: Refreshing all monitored playlists...
2021-09-22 08:37:04 [DEBUG] deemon.app.refresh: No new tracks have been added to playlist 'New Country'
2021-09-22 08:37:05 [DEBUG] deemon.app.refresh: New track 1486909902 detected on playlist 1306978785
2021-09-22 08:37:05 [INFO] deemon.app.refresh: Playlist 'Hot New Rock' has 1 new track(s)
2021-09-22 08:37:05 [DEBUG] deemon.app.refresh: Refreshing all monitored artists...
...
2021-09-22 08:39:29 [DEBUG] deemon.app.db: Database version 2.2
2021-09-22 08:39:29 [DEBUG] deemon.app.dmi: Initializing deemix library
2021-09-22 08:39:29 [DEBUG] deemon.app.dmi: deemix Config Path: /home/userName/.config/deemix
2021-09-22 08:39:29 [DEBUG] deemon.app.dmi: deemix Download Path: /home/userName/Music/
2021-09-22 08:39:29 [DEBUG] deemon.app.db: Database version 2.2
2021-09-22 08:39:29 [DEBUG] deemon.app.dmi: Looking for ARL...
2021-09-22 08:39:29 [DEBUG] deemon.app.dmi: ARL found in deemix config
2021-09-22 08:39:30 [DEBUG] deemon.app.dmi: ARL is valid
2021-09-22 08:39:30 [INFO] deemon.app.download: Sending 1 release(s) to deemix for download:
2021-09-22 08:39:30 [INFO] deemon.app.download: + Hot New Rock (playlist)...
2021-09-22 08:39:37 [INFO] deemix: [Hot New Rock track #1] Fallback to lower bitrate
2021-09-22 08:39:38 [INFO] deemix: [Hot New Rock track #2] Fallback to lower bitrate
...
2021-09-22 08:42:24 [INFO] deemon.app.download: Downloads complete!
```

**Comment by digitalec** [comment]
Confirmed and investigating

**Comment by digitalec** [comment]
Fixed and released in v1.3.2
